app/services/acceptable_verified_credential_adoption_service.rb

```diff
@@ -77,7 +78,29 @@ def verified_credential_at?
     user_avc&.acceptable_verified_credential_at
   end
 
-  def log_results(conversion_type)
+  def log_conversion_type_results(conversion_type)
     StatsD.increment("#{STATS_KEY}.#{conversion_type}.#{credential_type}")
   end
+
+  def record_adoption_email_trigger_event
+    CredentialAdoptionEmailRecord.create(
+      icn: user.icn,
+      email_address: user.email,
+      email_template_id: REACTIVATION_TEMPLATE_ID,
+      email_triggered_at: DateTime.now
+    )
+  end
+
+  def check_for_email_adoption_records
```

Comment:
Ooh, something I thought of here, I would probably query by ICN instead of email here. It's a safer field to use that's associated with the user

Reply:
I went back and forth on that too!  I see what you mean though...the email address could change while the ICN most likely will not.  The edge cases for the email switching during this limited test are likely limited BUT I'm happy to make the change if you feel strongly about it.

Just let me know :)

Reply:
yeah I'd recommend ICN, you'll have a way better time in general than trying to retrieve a user based off their email